Off-Top-App/off-top-flutter
# OT-ChangeFrontEndColor | Added the ability to change color themes

**What I Did:** 

- Added a new feature that allow a user to change their color themes from the settings page.

- Changed most of the app's color code to be dynamic rather than a set color. 

- Used temporary color palettes based on what I thought looked best, let me know if the colors should be changed. 

- Added colors to icons under the 'SELECT COLOR' menu so signify what colors will be chosen. 

**How To Test:**

- Prerequisites: You will need to run `flutter pub get` to pull the new packages that are used for the color picker. 

- You will also need  Zookeeper, Kafka, and Spring-boot servers running to log into the app.

- Go to the settings page, then open the 'SELECT COLOR' dropdown, then choose a new color.

- Test all the colors available under the dropdown. Make note of other pages of the app to see if the new color themes makes sense.
